api/src/main/java/org/openmrs/module/atomfeed/api/db/hibernate/HibernatingEventInterceptor.java

```diff
@@ -39,7 +39,7 @@
  */
 
 @Component
-public class HibernateEventInterceptor extends EmptyInterceptor {
+public class HibernatingEventInterceptor extends EmptyInterceptor {
```

Comment:
The class name should rather remain the same, change the bean name using something like:

@Component("atomfeed-hibernate-interceptor")

Reply:
Eventually you can change the classname to AtomFeedHibernateEventInterceptor